Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -266,8 +278,8 @@ - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray
     @"timestamp": @(CFAbsoluteTimeGetCurrent() * 1000.0) // in ms
   };
 
-  // Send event
-  if (_observingLocation) {
+  // Send event if observing, or if new location differs from last location
```

Comment:
The idea is that when you call "getLocation" you get a non stale location event. Maybe you don't want to observe the location (to conserve battery for instance), but occasionally get an updated location event. This will do that.